packages/beacon-node/test/spec/bls/bls.ts

```diff
@@ -31,7 +40,11 @@ export const testFnByType: Record<string, "skip" | ((data: any) => any)> = {
  */
 function aggregate_verify(input: {pubkeys: string[]; messages: string[]; signature: string}): boolean {
   const {pubkeys, messages, signature} = input;
-  return bls.verifyMultiple(pubkeys.map(fromHexString), messages.map(fromHexString), fromHexString(signature));
+  try {
```

Comment:
@wemeetagain I added this try/catch for the spec tests but am updating the base lib to not throw for this so it better meets the spec.